CHANGES.rst

```diff
@@ -1,7 +1,7 @@
 Changes
 =======
 
-Version 0.7.5 (UNRELEASED)
+Version 0.7.5 (2021-07-02)
 --------------------------
 - Changes workflow validation to display more granular output.
 - Fixes environment image validation not to test repetitively the same image.
```

Comment:
We may be missing two news items:

- Changes dependencies to unpin `six` so that client may be installed in more contexts.
- Fixes upload_to_server() Python API function to silently skip uploading in case of none-like inputs.

Reply:
One more cosmetic thing: please group fixes/changes messages together. We usually mention first "adds" (=new features), then "changes" (=feature improvements), then "fixes" (ending with bug fixes).